### ISSUE TITLE
libxmlHTMLDocument.swift showing Warnings in Xcode 11.4

### ISSUE THREAD

**Issue by ethanwa** [tid-kijyun/Kanna#227]
### Description:
libxmlHTMLDocument.swift showing the following Warnings in Xcode 11.4:

```
<my-project-path>/Pods/Kanna/Sources/Kanna/libxmlHTMLDocument.swift:222:30: Initialization of 'UnsafeRawPointer' results in a dangling pointer

<my-project-path>/Pods/Kanna/Sources/Kanna/libxmlHTMLDocument.swift:354:29: Initialization of 'UnsafeRawPointer' results in a dangling pointer
```
### Installation method:
- [ ] Carthage
- [x] CocoaPods
- [ ] Swift Package Manager
- [ ] Manually
- [ ] other: ()

**Comment by tid-kijyun** [comment]
Thank you for your report.
I've just released version 5.2.0 which fixes this issue.

**Comment by ethanwa** [comment]
Thank you, but now 5.2.0 is bringing up another warning:

https://github.com/tid-kijyun/Kanna/issues/231
